Circle.cpp

```diff
@@ -1,14 +1,28 @@
 #ifndef Circle_cpp
 #define Circle_cpp
+
 #include <iostream>
-#include "Draw.cpp"
+#include "Shape.cpp"
+#include "Area.cpp"
+
 using namespace std;
-class Circle : public Draw
+class Circle : public Shape
 {
 public:
     void drawShape() override
     {
-        cout << "circle" << endl;
+        cout << "circle" << endl; 
+    }
+    int inputAndCalculateArea() override
+    {
+        int Radius, area;
+        int PI = 3.14;
+        cout <<"Enter Circle Radius" <<endl;
+        cout <<"Enter Radius" <<endl;
+        cin >> Radius ;
+        area = PI*Radius*Radius;
+        cout << "Area = "  << area;
+        return 0;
```

Comment:
Why are we returning `0`? Isn't the function `inputAndCalculateArea` suppose to return area? Will it always remain `0` no matter user's input?